mlir/tools/mlir-rocm-runner/mlir-rocm-runner.cpp

```diff
@@ -33,9 +32,25 @@
 #include "mlir/InitAllDialects.h"
 #include "mlir/Transforms/DialectConversion.h"
 
+#include "mlir/Conversion/GPUCommon/GPUCommonPass.h"
+#include "mlir/Dialect/GPU/GPUDialect.h"
+#include "mlir/Dialect/GPU/Passes.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassManager.h"
```

Comment:
Can we check to make sure that these includes are all actually needed?

Reply:
Actually the GPUDialect.h and Pass.h are not needed.

Reply:
Yes, I did.